components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -437,7 +439,7 @@ public void deleteUser(String userId) throws NotFoundException, CharonException
     @Override
     public List<Object> listUsersWithGET(Node rootNode, Integer startIndex, Integer count, String sortBy,
                                          String sortOrder, String domainName, Map<String, Boolean> requiredAttributes)
-            throws CharonException, NotImplementedException {
+            throws CharonException, NotImplementedException, BadRequestException {
```

Comment:
Although `BadRequestException` the was not initially added it is available in the interface[1].

[1] https://github.com/wso2/charon/blob/master/modules/charon-core/src/main/java/org/wso2/charon3/core/extensions/UserManager.java#L67